carta/cpp/CartaLib/ICoordinateFormatter.h

```diff
@@ -56,6 +56,9 @@ class CoordinateFormatterInterface {
     /// enable printing of an axis
     virtual Me & enableAxis(int ind) = 0;
 
+    /// Set the axes the user will see (3 axis: x,y axes plus a hidden axis).
+    virtual void setDisplayAxes( std::vector<Carta::Lib::AxisInfo::KnownType> displayAxes ) = 0;
+
```

Comment:
@slovelan -- Is this still an issue?  Is it a major fix?  I think this PR is good to go if this is cleaned up.